### ISSUE TITLE
How create a triangle strip mesh?

### ISSUE THREAD

**Issue by ivan2007** [helix-toolkit/helix-toolkit#1526]
Hi, I need to create a triangle strip mesh.
I have positions, indices , texture coordinates and the texture (BC1 or BC3 format).
Which is the best way to create the mesh ?
I must use MeshBuilder or there is something better?

Thanks

**Comment by holance** [comment]
Helixtoolkit does not support to display triangle stripe directly. You may need to convert your mesh data to triangle list. You don't need to use MeshBuilder if you already have the mesh data. Just assign the data to MeshGeometry3D.

**Comment by ivan2007** [comment]
Thank you 
Following your suggestion I build the geometry of the mesh using MeshGeometry3D, while for the material I am using a DiffuseMaterial. To build the diffuse map I am using a TextureModel and an external library to read the data of my texture (the format is BC1 or BC3) and convert them in ARGB. Is there the support inside Helix for this type of format?

**Comment by holance** [comment]
Is your image DDS format? Helixtoolkit supports BC1 and BC3 under DDS.

**Comment by ivan2007** [comment]
No, I have only a stream of bytes. For the conversion I am using [this library](https://github.com/Nominom/BCnEncoder.NET).
